### PR TITLE
[IMPROVE] Filter markdown in notifications

### LINKED ISSUE #9979: [IMPROVEMENT] Remove all markdown from Push Notification
### Description:

It doesn't make any sense to have markdown tags in our push notification message, if we remove it would be much cleaner and easier do read.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.62.
* Operating System: Linux

### PR DESCRIPTION
**Pull request by c0dzilla**
@RocketChat/core 

Closes #9979 

This pr adds a filter to remove markdown tags from desktop/push notifications.

Example:

testingfilter

gives notification
filter_notification

### CODE REVIEW

**Comment by c0dzilla** [comment]
@sampaiodiego please review :)

**Comment by c0dzilla** [comment]
@rafaelks @sampaiodiego  is there anything else needed in this pr? Please have a look. Thanks :relaxed:

**Comment by c0dzilla** [comment]
@rodrigok please review :)

**Comment by rafaelks** [comment]
@rodrigok Milestone on the PR is 0.65 but issue is 0.66, maybe need to update it.

**Comment by rafaelks** [comment]
@rodrigok Do we need to ask @c0dzilla to do something on the PR in order to get review here?

**Comment by lorek123** [comment]
Any updates on this? This is something i would love to see this fixed.

**Comment by rodrigok** [comment]
@c0dzilla can you check the unit tests?

**Comment by c0dzilla** [comment]
@rodrigok fixed the tests.

File: app/markdown/lib/markdown.js [hunk 1]
```diff
@@ -89,7 +95,13 @@ const MarkdownMessage = (message) => {
 	return message;
 };
 
+const filterMarkdown = (message) => {
+	message = Markdown.filterMarkdownFromMessage(message);
+	return message;
```
Comment (rodrigok) [inline app/markdown/lib/markdown.js:100]:
```suggestion
	return Markdown.filterMarkdownFromMessage(message);
```

File: app/markdown/lib/markdown.js [hunk 2]
```diff
@@ -89,7 +95,13 @@ const MarkdownMessage = (message) => {
...
+};
+
 callbacks.add('renderMessage', MarkdownMessage, callbacks.priority.HIGH, 'markdown');
+callbacks.add('renderNotification', filterMarkdown, callbacks.priority.HIGH, 'markdown');
```
Comment (rodrigok) [inline app/markdown/lib/markdown.js:104]:
```suggestion
callbacks.add('renderNotification', filterMarkdown, callbacks.priority.HIGH, 'filter-markdown');
```

File: app/markdown/lib/markdown.js [hunk 3]
```diff
@@ -89,7 +95,12 @@ const MarkdownMessage = (message) => {
 	return message;
 };
 
+const filterMarkdown = (message) => {
+	return Markdown.filterMarkdownFromMessage(message);
+};
```
Comment (rodrigok) [inline app/markdown/lib/markdown.js:100]:
```suggestion
const filterMarkdown = (message) => Markdown.filterMarkdownFromMessage(message);
```

**Pull request merged**